src/test/java/edu/harvard/iq/dataverse/DatasetTest.java

```diff
@@ -49,5 +56,187 @@ public void testLocksManagement() {
         assertFalse( sut.isLocked() );
         
     }
+
+    // ****************************************************************************************
+    // The following tests test the isDeaccessioned method aiming for 100% prime path coverage.
+    // The following tool [1] was used to calculate the needed test paths.
+    // 11 test paths (= 11 test methods) are needed.
+    // 4 different dataset versions are needed:
+    // - DEACCESSIONED
+    // - DRAFT
+    // - RELEASED
+    // - non of the above (= ARCHIVED)
+    //
+    // See details here: https://github.com/IQSS/dataverse/pull/5703
+    //
+    // [1] https://cs.gmu.edu:8443/offutt/coverage/GraphCoverage
```

Comment:
I left a comment on @nikzaugg 's commit that applies here. Looking at this tool, I do not understand how it was used with our codebase.

See this comment for info on how to proceed: https://github.com/IQSS/dataverse/pull/5705#discussion_r271425094

Reply:
I replied to this matter on the PR of @nikzaugg here: https://github.com/IQSS/dataverse/pull/5705#discussion_r271706583

Reply:
Please remove the references to the graphing tool from the tests and write up a basic overview of the cases tested (its fine if it doesn't cover "all the edges of the graph"). I'm glad its been useful to understand and write tests cases, but its too much of a maintenance/code-rot burden to have info about it live on in the code.

Reply:
I removed the reference to the tool and added a short description in (b378bca).